### DISCUSSION TITLE
How can I delete an unsupported cluster capability from cluster version [4.11]?

### DISCUSSION THREAD

**Discussion by tothlali** [okd-project/okd#1516] [Q&A]
OKD 4.11 upgrade stucked:
![Screenshot_20230222_145819](https://user-images.githubusercontent.com/123624694/220645391-c15161a4-0a08-49bb-961a-c74dbb724f2e.png)

Problem with version:
![Screenshot_20230222_150828](https://user-images.githubusercontent.com/123624694/220645968-5f64c3ba-9cda-407b-b6df-e222d55d2f65.png)

**Comment by vrutkovs** [comment]
Please attach (or upload to the public file sharing service) [must-gather](https://docs.okd.io/latest/support/gathering-cluster-data.html#support_gathering_data_gathering-cluster-data) archive

**Comment by tothlali** [comment]
Must-gather:
[https://www.dropbox.com/s/n6vs9zl7esrpvhl/must-gather.local.1164074284303162154.tar.gz?dl=0](url)

btw failed updating to 4.12 occurred earlier :
[https://github.com/okd-project/okd/discussions/1472#discussioncomment-4784988](url)

**Reply by vrutkovs** [reply]
That must-gather is complete. Seems your API server is malfunctioning?

**Reply by tothlali** [reply]
Don't think so. I think that updating can not finish (however all cluster operator OK) because cluster version can not be modified (because of unsupported capabilities). I mean everything is working, just cluster version is "read-only", so can not start a new update toward 4.12. 
The unsupported capabilities invaded the cluster version at the failed 4.12 update (see my second link) - I think.
